packages/chartjs-annotation/chartjs-annotation.0.2.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "sashayanin@gmail.com"
+authors: ["Alexander Yanin"]
+homepage: "https://github.com/monstasat/chartjs-ocaml"
+dev-repo: "git+https://github.com/monstasat/chartjs-ocaml.git"
+bug-reports: "https://github.com/monstasat/chartjs-ocaml/issues"
+license: "MIT"
+        
+synopsis: "OCaml bindigns for Chart.js annotation plugin"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
+  "js_of_ocaml"
+  "js_of_ocaml-ppx"
+  "chartjs"
```

Comment:
```suggestion
  "chartjs" {= version}
```
Are you sure there is no minimum and maxium version for this dependency? I'd recommend to use `= version` to match every releases together.